### ISSUE TITLE
In graph editor, model visually flipped and annotons became uneditable

### ISSUE THREAD

**Issue by suzialeksander** [geneontology/noctua#551]
I added individual functions with the graphical editor, have been trying to add relations but the annotons keep "jumping" when edited for content or edges.  Used Cytoscape tool to corral all the annotons in a reasonably workable area, came back to the model- and all my annotons are flipped.  The edges are all in a separate corner of the canvas. 

Also, "zoom out" zoomed in, and "zoom in" went out.  

Logging out & back in righted the annotons, but many were outside of the canvas until I went back to Cytoscape viewer (changed nothing) and returned.  They are now behaving slightly better.

Is this a bug or a gross misuse of the tools?

gomodel:5a7e68a100000920 
Thanks
Suzi A

![noctuaupsidedown](https://user-images.githubusercontent.com/13735608/38153918-0b26411e-3424-11e8-852c-49144ff48cb0.jpeg)
![noctuarelations](https://user-images.githubusercontent.com/13735608/38153919-0b3b2fac-3424-11e8-81ad-378438f8dd08.jpeg)

**Comment by kltm** [comment]
I have never seen anything like that before; although some things are "similar", in the way that a match is like a bonfire.

What browser/OS are you using and can you recreate this from a hard refresh from the landing page with a series of steps?

(Note to self: possibly related to #95 and/or #360).

**Comment by kltm** [comment]
Just...wow...

**Comment by suzialeksander** [comment]
No, I so far haven't been able to reproduce, but I will be working on this model for a while I'm sure. It has "jumpy" since yesterday. This is probably posted for ~~amusement~~ future reference.  Using FF 59.0.2 on OS 10.2.6, should I switch to Chrome or keep using FF to see if this (unlikely) happens again?

**Comment by kltm** [comment]
Uhhh...
FF should be fine--that's generally what is developed on. Either should be fine though.
I would note one thing: either you have an enormous screen (congrats) or you are editing with the zoom levels adjusted? The former is not interesting, but the latter is a possibility as the code was never designed to work with those adjustments in editing--they were just quick hacks for public viewing.
If this spreads to other models or other users report this and I cannot reproduce, I may need to get a hold of your machine.

**Comment by kltm** [comment]
BTW, I was poking around on your model, and nothing weird yet...

**Comment by suzialeksander** [comment]
Good to know.  I'll reset the zoom to 100% and let you know if anything like this reoccurs.

**Comment by kltm** [comment]
Okay, it's been a while. I think we can just chalk this up to your browser being wacky.
If it occurs again, please go ahead an open this back up.

**Comment by kltm** [comment]
@ValWood has a new instance of this, with: https://github.com/geneontology/noctua/issues/702

**Comment by vanaukenk** [comment]
Closing for now.  If we want to try to address this, we can re-open.
